demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -997,6 +1000,84 @@ def get_file_description(path, file_type) -> str:
 
 
 def update_api_modules_dependents_rn(
+    pre_release: bool,
+    update_type: Union[str, None],
+    added: Union[list, set],
+    modified: Union[list, set],
```

Comment:
replace `Union[list,set]` with `Iterable[str]` (if the values inside those are strings)